lib/libc/Kconfig

```diff
@@ -239,7 +239,20 @@ config LIBC_TZDIR
 	string "zoneinfo directory path"
 	default "/etc/zoneinfo"
 
-endif
+config LIBC_DOWNLOAD_ZONEINFO
+	bool "Download TZ database"
+	default y
+	---help---
+		Download and build the TZ/Olson database.
+
+config LIBC_ZONEINFO_ROMFS
+	bool "Build ROMFS filesystem"
+	default n
+	depends on FS_ROMFS
```

Comment:
How about changing to `select`?

Reply:
FS_ROMFS depends on (!DISABLE_MOUNTPOINT && MTD_FTL). So I think (depends on FS_ROMFS) is good.